docking.js

```diff
@@ -1375,6 +1376,67 @@ const DockedDash = new Lang.Class({
             else
                 return false;
         }
+    },
+
+    /**
+     * Isolate overview to open new windows for inactive apps
+     */
+    _optionalWorkspaceIsolation: function() {
+
+        let OverviewLabel = 'optionalWorkspaceIsolation';
```

Comment:
We don't need that really. I kept them separate in case you wanted to create two different functions, that way it was easier to spread the code into 2.

Patch [fe2b3151fc2fb545b62121c67f97a8b85167d26a] fixes that!